src/components/counter.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, useState } from "react";
+
+type CounterProps = {
```

Comment:
Just TS advise: type aliases should be used only if you have some types operation like union/extractions. Since your type is just a props description I advise to use interfaces.